contents/handbook/strategy/business-model.md

```diff
@@ -29,12 +25,14 @@ If the wider community contributes a new feature that isn't already a source-ava
 
 When PostHog makes a new feature, we ask ourselves two questions:
 
-1. Who is the likely type buyer of this feature?
+1. Is this feature for Creation, Collaboration or Compliance?
 1. Would this feature help more users find and use PostHog?
 
-If the likely buyer is an individual contributor, the feature will be open source. Otherwise, if the likely buyer is a manager, director or executive, it will be source available. The exception to this is if the feature will significantly help the community to increase. For example, initially we planned "multiple users" as a feature for the source-available version. However, we decided that having multiple users would help the community to grow, which benefits everyone disproportionately.
+If the likely use case is use by an individual contributor, the feature will be open source. Otherwise, it will be source available. The exception to this is if the feature will significantly help the community to increase. For example, initially we planned "multiple users" as a feature for the source-available version. However, we decided that having multiple users would help the community to grow, which benefits everyone disproportionately.
```

Comment:
> If the likely use case is use by an individual contributor, the feature will be open source

This differs to "All new major features will be paid only by default" [here](https://posthog.com/handbook/product/paid-features) which I think we are more aligned with.

How about

```suggestion
### Creation, Collaboration or Compliance**
- Creation: if the feature improves our existing open source products and is focused on helping individual contributors then it will be open source. If it's a major new feature or product line it should default to being a paid feature.
- Collaboration: if the feature helps teams to collaborate on Posthog it should be a paid feature. The exception to this is if the feature will significantly help the community to increase. For example, initially we planned "multiple users" as a feature for the source-available version. However, we decided that having multiple users would help the community to grow, which benefits everyone disproportionately.
- Compliance: if the feature is disproportionately used by enterprises for compliance it should be an enterprise feature.
```